docs/.custom_wordlist.txt

```diff
@@ -16,13 +16,16 @@ deployable
 Disposability
 downscaled
 entrypoint
+ephemeral
 ExpressJS
 FastAPI
 filesystem
 FQDN
+Gradle
 Gunicorn
 init
 Jinja
+kubectl
```

Comment:
I don't quite see this word in the PR, if this is an issue, I believe this can be avoided by putting `kubectl` in the ``code syntax``